src/pages/gift/about/FAQ.js

```diff
@@ -22,10 +22,10 @@ export default function FAQ() {
     </p>
   );
   const a2 =
-    'Polkadot Gifts lets you generate a unique hash that functions as a voucher to claim an amount of DOT. Send the unique gift secret to your friend or family member. They are guided through a simple account creation process and reveal the gift secret to the Polkadot network to claim their DOTs into their account.';
+    'Polkadot Gifts is powered by on chain logic provided by the Polkadot Network. This website is completely stateless and does not track any sensitive information. All interactions are done between you and the Polkadot blockchain, keeping your account, your funds, and the gift claiming process secure.';
```

Comment:
I like this explanatory bit about not tracking. Still for someone new, we need a basic explanation of why you want to use this app in the first place